src/System.Net.Http/ref/System.Net.Http.csproj

```diff
@@ -3,6 +3,9 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <ProjectGuid>{132BF813-FC40-4D39-8B6F-E55D7633F0ED}</ProjectGuid>
+    <!-- in most cases the desktop configuration of a contract is just for APICompat,
+         however in this case it is the actual ref we want to ship -->
+    <IncludeDesktopRefInPackage>true</IncludeDesktopRefInPackage>
```

Comment:
What does this mean w.r.t. the 8 new properties we added to HttpClientHandler in System.Net.Http?  Will those 8 new properties disappear from the 'ref' because now you are using the references from .NET Framework (Desktop) which don't have those 8 new properties?  Or are these changes still preserving the 'ref' from src/System.Net.Http/ref/System.Net.Http.cs in .NET Core and putting that into the System.Net.Http package?